zulip_botserver/zulip_botserver/server.py

```diff
@@ -68,13 +70,22 @@ def parse_config_file(config_file_path: str) -> configparser.ConfigParser:
     parser.read(config_file_path)
     return parser
 
+def load_module_from_file(file_path: str) -> ModuleType:
+    # Wrapper around importutil; see https://stackoverflow.com/a/67692/3909240.
+    spec = importlib.util.spec_from_file_location("custom_bot_module", file_path)
+    lib_module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(lib_module)  # type: ignore # mypy thinks the `spec` is None in Optional[Loader].
```

Comment:
This function is called from inside a `try-except` block, and we print:

```py
except ImportError:
            error_message = ("Error: Bot \"{}\" doesn't exist. Please make sure "
                             "you have set up the botserverrc file correctly.\n".format(bot))
```

Reply:
(A slightly related issue in mypy: https://github.com/python/mypy/issues/2420)

I'm working on tests for this, that should add additional checking for our purposes. It would be trivial to add a simple `if not spec: raise ImportError()`, though.